src/utils/formatValue.js

```diff
@@ -36,17 +36,17 @@ const formatValue = (formatter, value) => {
   if (value === null || value === undefined) return null;
   switch (formatter) {
     case 'date':
-      return moment(value).utc().format(dateFormat);
+      return moment.utc(value).format(dateFormat);
```

Comment:
- [ ] Ensure whatever we have here, it matches https://github.com/Zenlytic/zenlytic/pull/2483/files.